src/FileServiceManager/FileService.Tests/json1/FileSystemConfig.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
please consider renaming json1 folder name to something like Assets

Reply:
done